### PR TITLE
Release 2.2.0-beta1

### PR DESCRIPTION
**Pull request by niik**
## Overview

This is the release PR for the first beta release of the 2.2.0 series.

## Description

The main addition here is the onboarding tutorial.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.2.0-beta1": [
+      "[New] Interactive tutorial for new users to become productive using Git and GitHub more quickly",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Interactive tutorial for new users to become productive using Git and GitHub more quickly - #8148 #8149",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.2.0-beta1": [
+      "[New] Interactive tutorial for new users to become productive using Git and GitHub more quickly - #8148 #8149",
+      "[Fixed] Commit summaries with comma delimited issues are parsed. Thanks @say25!",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Commit summaries with comma delimited issues are parsed - #8162. Thanks @say25!",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.2.0-beta1": [
+      "[New] Interactive tutorial for new users to become productive using Git and GitHub more quickly - #8148 #8149",
+      "[Fixed] Commit summaries with comma delimited issues are parsed - #8162. Thanks @say25!",
+      "[Fixed] Support CodeRunner as an external editor - #8091. Thanks @ns-ccollins!",
```
Comment (say25) [inline changelog.json]:
```suggestion
      "[New] Support CodeRunner as an external editor - #8091. Thanks @ns-ccollins!",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] Support VSCodium as an external editor - #8000. Thanks @Rexogamer!",
```
Comment (say25) [inline changelog.json]:
```suggestion
      "[New] Support VSCodium as an external editor - #8000. Thanks @Rexogamer!",
```

**Comment by outofambit** [comment]
@billygriffin can i get a ✅ so I can merge this? don't want to let this sit.

**Comment by outofambit** [comment]
(got a verbal ✅ from @billygriffin)

**Pull request merged**